src/atoms/link.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Link from '@mui/material/Link';
+
+interface UnderlineLinkProps {
+    children: string;
+    linkType: 'primary' | 'secondary' | 'default';
```

Comment:
if "default" is not being use, you could remove it, and maybe have "primary" as default value

Reply:
bueno ingeniero, voy a poner que si no pasan linkType quede por defecto el 'default'